bot/extensions/threads_cog.py

```diff
@@ -120,6 +133,47 @@ def cog_load(self):
             )
         )
 
+        # Runs reminders everyday at 12:30
+        self.jobs.append(
+            self.bot.scheduler.add_job(
+                self.daily_reminder, "cron", hour=12, minute=30, timezone=self.timezone
+            )
+        )
+
+    async def daily_reminder(self):
+        """Send a daily reminder for active threads."""
+        info("Posting daily threads's reminder")
+
+        embed = Embed(
+            color=self.bot.default_color,
+            title="🔔 Daily Reminder",
+            description="Join the discussion in the latest active threads:",
+        )
+
+        if threads := Thread.all():
+            for thread in threads:
+                if (
+                    hasattr(thread, "latest_thread")
```

Comment:
I don't think you need `hasattr`. If `thread.latest_thread` is None, it will evaluate to False. also, related to my comment above, you don't need the daily reminder or event do all this manually.